doc/portfile.7

```diff
@@ -946,6 +946,34 @@ to
 .br
 .Sy Example:
 .Dl use_bzip2 yes
+.It Ic use_lzip
+Use lzip.
+.br
+Sets
+.Ic extract.suffix
+to
+.Dq .tar.lz .
+.br
+Sets
+.Ic extract.cmd
+to
+.Dq lzip .
+.br
+Sets
+.Ic extract.pre_args
+to
+.Dq -dc .
+.br
+Sets
+.Ic extract.post_args
+to
+.Dq | tar -xf - .
```

Comment:
Isn't this the default anyways for `extract.post_args` without use_zlip?

https://github.com/macports/macports-base/blob/1d6588d178b251c86abc37cbea5ab580e2b85fd1/src/port1.0/portextract.tcl#L58

Reply:
Looks good otherwise